### ISSUE TITLE
Filter: Display Objects with filter are getting invisible after hours of run

### ISSUE THREAD

**Issue by Adolio** [Gamua/Starling-Framework#1071]
Hi Daniel,

I recently noticed that all Display Objects with filters are not rendered anymore after several hours of the app running. When this occurred, there were no (uncaught) error thrown. The build was done using AIR 32 on Windows.

Do you know if someone has experienced the same issue in the past?

I would say that it sounds like a GPU out of memory issue... but unfortunately I don't know what was the memory state when it occurred. I have now added debug logs in few places to track that if I can reproduce.

**Comment by PrimaryFeather** [comment]
Oh wow — no, I haven't heard of this issue before!
What does the stats display say, i.e. does the GPU memory consumption rise over time, or does it stay the same? (It sounds like it should rise, but the question is if those stats are correct.)

On what platform/OS did you notice this?

**Comment by Adolio** [comment]
This issue happened again on my side on another machine after approximately one hour of play (still on `AIR 32` with `Starling 2.6.1` on `Windows 10` under `DirectX11 (Enhanced)`). This time it wasn't a standalone build, but it ran through `adl.exe (32-bit)`.

GPU & standard memory looks good. No warnings & no errors from both Starling & AIR... 🤔

From my logs:
```
[2020-06-02 18:02:59.461] [INFO ] [Main] Current system memory: 128.84 MB
[2020-06-02 18:02:59.461] [INFO ] [Main] Current GPU memory: 592.20 MB
```

That's a really weird... again only the filters/shaders are broken (Display Objects getting invisible).

**Comment by Adolio** [comment]
Note that there is high chance that this issue is related to: https://github.com/Gamua/Adobe-Runtime-Support/issues/38.

**Comment by Adolio** [comment]
This issue is now fixed in `AIR 33.1.1.217` 🥳

Thanks Harman AIR Team!
